docs/02-guides/server-side-events/create-an-integration.mdx

```diff
@@ -7,7 +7,135 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+This guide will help you to create an integration that will dispatch events to external services.
+
+We will create a new integration to dispatch events to [Wehook.cool](https://webhook.cool/).
+
+:::info
+
+Front-Commerce already provide a Webhook integration. This guide will show you how to create an integration from scratch. And will explain you the basics of integrations so you can write your own
```

Comment:
```suggestion
Front-Commerce already provides a Webhook integration. This guide will show you how to create an integration from scratch. And will explain you the basics of integrations so you can write your own.
```

Reply:
I would remove the "Front-Commerce already provides a Webhook integration." all together. it does not make much sense unless you link to the Webhook integration with documentation.